src/lib/Components/Inbox/Conversations/Previews/PDFPreview.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import { TouchableHighlight } from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../../../../data/colors"
+import fonts from "../../../../../data/fonts"
+import OpaqueImageView from "../../../OpaqueImageView"
+
+const Container = styled.View`
+  borderWidth: 1
+  borderColor: ${colors["gray-regular"]}
+  flexDirection: row
```

Comment:
I believe that kebab-case was requested by others in another PR, but don’t have a link to that offhand.